### PR TITLE
Taking a few fixes to master

### PR DESCRIPTION
**Pull request by sorosoro2**
- Refreshing on pods details is not taking the pod uid into consideration
- Refresh on orphan pods is not working
- sub header on services details is incorrect

### CODE REVIEW

File: src/WebUI/Pods/PodLog.tsx [hunk 1]
```diff
@@ -10,8 +10,11 @@ import { KubeSummary } from "../Common/KubeSummary";
 import * as Resources from "../Resources";
 import { PodContentReader } from "./PodContentReader";
 import { IPodRightPanelProps } from "./PodsRightPanel";
+import { V1Pod } from "@kubernetes/client-node";
 
-export interface IPodLogProps extends IPodRightPanelProps { }
+export interface IPodLogProps extends IPodRightPanelProps {
+    pod: V1Pod;
```
Comment (madhuv-msft) [inline src/WebUI/Pods/PodLog.tsx]: add this to IPodRightPanelProps

File: src/WebUI/Pods/PodsRightPanel.tsx [hunk 2]
```diff
@@ -73,43 +110,45 @@ export class PodsRightPanel extends BaseComponent<IPodRightPanelProps, IPodsRigh
...
+                        {this._getPageContent()}
+                    </div>
+                </Page>)
+                : <Spinner className={"flex-grow flex-center"} label={Resources.LoadingText} />
         );
     }
 
     private _getHeader(): JSX.Element | null {
-        return (
+        return this.state.pod ? (
```
Comment (madhuv-msft) [inline src/WebUI/Pods/PodsRightPanel.tsx]: do we need these changes of pod check? as we are not going to call these untill pod is available?

File: src/WebUI/Pods/PodLog.tsx (hunk 1 above)
Comment (sorosoro2) [reply src/WebUI/Pods/PodLog.tsx]: The base class has this, but there I have to keep an option of this being undefined for refresh scenario, but these components should have it as not null

Comment (madhuv-msft) [reply src/WebUI/Pods/PodLog.tsx]: would you add a comment, so we do not lose track of this fact please.

**Pull request merged**